### PR TITLE
12972+ Localized debug step PAUSED ON

### LINKED ISSUE #12972: Debug status is not fully localized
### Bug Description:
Debug status does not have localization of phrases "Paused on breakpoint" and so on.

### Steps to Reproduce:
https://github.com/eclipse-theia/theia/blob/525b3b11ebc42adc986acb0e5c1b33513e5e4ad5/packages/debug/src/browser/model/debug-thread.tsx#L233

Reason could be any. It is enumerated in vscode debugprotocol

```
reason: 'step' | 'breakpoint' | 'exception' | 'pause' | 'entry' | 'goto' | 'function breakpoint' | 'data breakpoint' | 'instruction breakpoint' | string;
```

### Additional Information

- Operating System: any
- Theia Version: the latest

### PR DESCRIPTION
**Pull request by Zebsterpasha**
#### What it does

Fixes #12972 

#### How to test

Just check the localization keys

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by Zebsterpasha** [comment]
@vince-fugnitto @msujew Hello) Please, take a look at this request, it might improve localization) Thanks in advance)

File: packages/debug/src/browser/model/debug-thread.tsx [hunk 1]
```diff
@@ -230,11 +230,44 @@ export class DebugThread extends DebugThreadData implements TreeElement {
...
+            case 'breakpoint':
+                return nls.localize('theia/debug/entry', 'entry');
+            case 'breakpoint':
+                return nls.localize('theia/debug/goto', 'goto');
+            case 'breakpoint':
+                return nls.localize('theia/debug/function breakpoint', 'function breakpoint');
+             case 'breakpoint':
+                return nls.localize('theia/debug/data breakpoint', 'data breakpoint');
+            case 'breakpoint':
+                return nls.localize('theia/debug/instruction breakpoint', 'instruction breakpoint');
```
Comment (msujew) [inline packages/debug/src/browser/model/debug-thread.tsx:267]: Issue: These cases are all the same.

File: packages/debug/src/browser/model/debug-thread.tsx [hunk 2]
```diff
@@ -230,11 +230,44 @@ export class DebugThread extends DebugThreadData implements TreeElement {
...
+            case 'breakpoint':
+                return nls.localize('theia/debug/exception', 'exception');
+            case 'breakpoint':
+                return nls.localize('theia/debug/pause', 'pause');
+            case 'breakpoint':
+                return nls.localize('theia/debug/entry', 'entry');
+            case 'breakpoint':
+                return nls.localize('theia/debug/goto', 'goto');
+            case 'breakpoint':
+                return nls.localize('theia/debug/function breakpoint', 'function breakpoint');
```
Comment (msujew) [inline packages/debug/src/browser/model/debug-thread.tsx:263]: Suggestion: Don't use whitespace in translation keys
```suggestion
                return nls.localize('theia/debug/functionBreakpoint', 'function breakpoint');
```

File: packages/debug/src/browser/model/debug-thread.tsx [hunk 3]
```diff
@@ -230,11 +230,44 @@ export class DebugThread extends DebugThreadData implements TreeElement {
...
-        </div>;
+        const localizedReason = this.getlocalizedReason(reason);
+
+        const status = this.stoppedDetails
+            ? reason
+                ? nls.localizeByDefault('Paused on {0}', localizedReason)
+                : nls.localizeByDefault('Paused')
+            : nls.localizeByDefault('Running');
+        return (
+            <div className="theia-debug-thread" title="Thread">
```
Comment (msujew) [inline packages/debug/src/browser/model/debug-thread.tsx:241]: Suggestion: You can localize `Thread` correctly as well while you're at it.
```suggestion
            <div className="theia-debug-thread" title={nls.localizeByDefault('Session')}>
```

File: packages/debug/src/browser/model/debug-thread.tsx [hunk 4]
```diff
@@ -230,11 +230,44 @@ export class DebugThread extends DebugThreadData implements TreeElement {
...
+                <span className="label">{this.raw.name}</span>
+                <span className="status">{status}</span>
+            </div>
+        );
+    }
+
+    private getlocalizedReason(reason: string | undefined): string {
```
Comment (msujew) [inline packages/debug/src/browser/model/debug-thread.tsx:248]: Suggestion: Please don't use `private` methods - this prevents adopters to override the behavior. Use `protected` instead.

File: packages/debug/src/browser/model/debug-thread.tsx (hunk 1 above)
Comment (Zebsterpasha) [reply packages/debug/src/browser/model/debug-thread.tsx]: My bad, i've just fixed it)

**Review by msujew — APPROVED** [review]
Thanks, looks good to me 👍

**Pull request merged**